src/loki_alert_rules/log_volume.yaml

```diff
@@ -0,0 +1,10 @@
+groups:
+  - name: high-log-volume
+    rules:
+      - alert: HighLogVolume
+        expr: |
+          count_over_time(({%%juju_topology%%})[30s]) > 100
```

Comment:
Do we need this alert?
Iirc, whenever traefik restarts we get more than 100 already.